lib/archethic/contracts/interpreter/library/common/chain_impl.ex

```diff
@@ -18,4 +20,18 @@ defmodule Archethic.Contracts.Interpreter.Library.Common.ChainImpl do
   defdelegate get_genesis_public_key(public_key),
     to: Legacy.Library,
     as: :get_genesis_public_key
+
+  @impl Archethic.Contracts.Interpreter.Library.Common.Chain
+  def get_transaction(address) do
+    address
+    |> UtilsInterpreter.get_address("Chain.get_transaction/1")
+    |> Archethic.search_transaction()
+    |> then(fn
+      {:ok, tx} ->
+        Constants.from_transaction(tx)
+
+      {:error, _} ->
+        nil
```

Comment:
Currently other `Chain` functions raise an error if the result is invalid. In this case you are returning nil which is not coherent with other functions.
We should have an unified way to return error.

Reply:
- [ ] Update all `Chain` functions to return `nil` instead of crashing

Reply:
- [ ] expect instead of stub